scripts/db-heal-test.py

```diff
@@ -0,0 +1,231 @@
+import os
+import subprocess
+import tempfile
+import time
+import shutil
+from pathlib import Path
+import sys
+import flag
+
+# --- Dynamic variables --- #
+aida_path_flag = flag.string("aida", "", "Path to Aida root.")
+aida_db_path_flag = flag.string("aida_db", "", "Path to AidaDB.")
+tmp_path_flag = flag.string("tmp", "", "Path to tmp dir.")
+
+number_of_iterations_flag = flag.int("iter", 1000, "Number of iterations.")
+window_flag = flag.int("window", 120, "How often will program get terminated (in seconds).")
+checkpoint_granularity_flag = flag.int("cp_granularity", 2000, "How often will Carmen create checkpoints.")
+flag.parse()
+
+aida_path = aida_path_flag.value
+aida_db_path = aida_db_path_flag.value
+tmp_path = tmp_path_flag.value
+
+number_of_iterations = number_of_iterations_flag.value
+window = window_flag.value
+checkpoint_granularity = checkpoint_granularity_flag.value
+
+# Mark first checkpoint
+latest_checkpoint = checkpoint_granularity
+
+if aida_path == "":
+    print("please set Aida using --aida")
+    exit(1)
+if aida_db_path == "":
+    print("please set AidaDB using --aidadb")
```

Comment:
this took me some time:  I would somehow naturally expect `--aida-db`, but here it says to use `--aidadb`, actually it works with `--aida_db`

I would suggest to unify to  `--aida-db`
* without separator in the middle it is difficult to read
* underscore I think is not very commonly used as an argument.

Reply:
also for other parameters, I think underscore does not seem natural being used as a program parameter.

Reply:
unfortunately python (or this library) does not allow dashes in flag names.

Reply:
could you use this library: https://docs.python.org/dev/library/argparse.html#dest

it seems to support this.  But most importantly, it seems to be standard, and already installed with python. The current library I had to install with some hack bypassing pip sandboxing